modules/server/src/main/resources/db/migration/V1__add_protocols_table.sql

```diff
@@ -1,3 +1,4 @@
 CREATE TABLE protocols (
-    id VARCHAR PRIMARY KEY
-)
+    id VARCHAR PRIMARY KEY,
+    idl_name VARCHAR NOT NULL
```

Comment:
We should define `idl_name` as an [enum_type](https://www.postgresql.org/docs/current/datatype-enum.html)

Reply:
Done